tests/test.py

```diff
@@ -69,6 +76,14 @@ def test_setup_oauth(self, session_mock, json_mock):
             string_headers = headers.read()
             self.yt_oauth = YTMusic(string_headers)
 
+    def test_alt_oauth(self):
+        # ensure client works/ignores alt if browser credentials passed as auth
+        self.assertFalse(self.yt_alt_oauth.is_alt_oauth)
+        # oauth token dict entry and alt
+        self.yt_alt_oauth = YTMusic(json.loads(config['auth']['oauth_token']),
+                                    oauth_credentials=alt_oauth_creds)
+        self.assertTrue(self.yt_alt_oauth.is_alt_oauth)
```

Comment:
This assert is still failing for me.  The key `expires_at` is missing when `is_oauth` is called, therefore it never goes into the if block to set `is_alt_oauth`